pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -182,25 +182,15 @@ func (m *kubeGenericRuntimeManager) getKubeletSandboxes(all bool) ([]*runtimeApi
 // determinePodSandboxIP determines the IP address of the given pod sandbox.
 // TODO: remove determinePodSandboxIP after networking is delegated to the container runtime.
```

Comment:
yeah, the comment isn't valid because this function does 0 "work" today, it just takes the status returned by the plugin and pulls out the ip (+ some validation that it is an actual ip).